.github/workflows/deploy-zap-dev.yaml

```diff
@@ -54,9 +54,12 @@ jobs:
 
       - name: Run migrations
         run: make aws-run-migrator
-
+      
       - name: Deploy All
         run: make aws-deploy-all
+      
+      - name: Deploy Dev Clear DB Lambda
```

Comment:
Why do we want to do this step here?

Reply:
This lambda is only deployed dev and so it is not included in the "deploy all" Makefile command.

Reply:
Are we sure we don't need this lambda available on test, only dev?

I'm sorry for having parallel conversations in Jira/Teams/PR. The ticket is also out of date, whatever we decide on this then let's get it updated there.

Reply:
I went with dev only since we have the sync job enabled on test